assets/styles/styles.scss

```diff
@@ -1,6 +1,19 @@
 // basic test file
 
+// colors
+$yellow: #ffff00;
+$blue: #0000ff;
+
 // broken on purpose!
+// div {
+//   border: 2px solid yellow;
+//   color: black;
+//   background-color: blue;
+// }
+
+// not broken\
 div {
-  border: 2px solid yellow;
+  border: 2px solid $yellow;
+  color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.